docker-compose.yml

```diff
@@ -36,6 +36,19 @@ x-github-pat: &github-pat
   GITHUB_PAT: ${GITHUB_PAT}
 
 services:
+  prefect_server:
+    image: prefecthq/prefect:3-latest
+    command: prefect server start --host 0.0.0.0
+    ports:
+      - ${PREFECT_SERVER_API_PORT:-4200}:${PREFECT_SERVER_API_PORT:-4200}
+    environment:
+      PREFECT_SERVER_API_HOST: ${PREFECT_SERVER_API_HOST:-localhost}
+      PREFECT_SERVER_API_PORT: ${PREFECT_SERVER_API_PORT:-4200}
+      PREFECT_SERVER_API_AUTH_STRING: ${PREFECT_SERVER_API_AUTH_STRING:-admin:pass}
+      PREFECT_API_AUTH_STRING: ${PREFECT_API_AUTH_STRING:-admin:pass}
```

Comment:
Oopf I'm not sure about having a default login, can we just make it fail if you dont define it so that we're forced to set one. Or fail in production if its the default?